playbooks/roles/backups/tasks/main.yml

```
@@ -147,3 +147,18 @@
     - service: mysql
       enabled: "{{ BACKUPS_MYSQL }}"
   tags: ['backups']
+
+# one of the system-wide pip installs that runs at some point previous
+# to install libraries upgrades urllib3 to a version that breaks
+# ansible on subsequent deploys.
+# See: https://appsembler.atlassian.net/wiki/spaces/ED/pages/390004931/Failed+to+install+repo+key
+# the *correct* fix for this would be to have our backups script and
+# its dependencies installed into a virtualenv so it doesn't conflict
+# with system-wide libraries. That's a large change though
+# so for now, as a workaround, we just downgrade urllib3 back to a known safe version
+- name: pin urllib3 back down to a version that doesn't break ansible
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items:
+    - {name: urllib3, version: 1.13.1}
+  when: BACKUPS_PROVIDER == 'gs'
```

In case we forgot to remove this in Juniper.

```suggestion
  when: 
    - BACKUPS_PROVIDER == 'gs'
    - BACKUPS_URLLIB3_DOWNGRADE | default(False)
```